tests/common.go

```diff
@@ -191,6 +194,11 @@ var (
 	GlobalGkeSecretString  string
 )
 
+var (
+	NamespaceAppWithDataMap    = make(map[string][]appDriver.ApplicationDriver)
+	IsReplacePolicySetToDelete = false // To check if the policy in the test is set to delete - Skip data continuity validation in this case
```

Comment:
In which case are going to set this to true?

Reply:
Under ValidateRestore if theRestore.ReplacePolicy != api.ReplacePolicy_Retain